Directory.Build.targets

```diff
@@ -29,6 +29,9 @@
 
     <!-- Legacy targets do not support attributes for a nullable context thus suppressing null check warnings -->
     <NoWarn Condition="'$(TargetFramework)' == 'net462' or '$(TargetFramework)' == 'netstandard2.0'">$(NoWarn);CA1062</NoWarn>
+
+    <!-- Stop complaining about enabling /doc for improve analysis trimming info -->
```

Comment:
I'm not sure I understand this comment. We do care about analysis of trimming info, right? Our assemblies should be trim safe.

Reply:
This is about reference trimming, not AOT-related trimming.

This warning is telling me to compile assemblies with /doc in order to get a better quality of analysis. We already compile all the product assemblies with /doc so the warning is coming out for all the test and benchmark assemblies, for which we don't use /doc and don't really care about superfluous references anyway.

Reply:
I moved the suppression to be in the various subdirectories that require it rather than in this file, and I improved the comment.